pages/1.10/administering-clusters/convert-agent-type/index.md

```diff
@@ -16,15 +16,17 @@ You can convert agent nodes to public or private for an existing DC/OS cluster.
 Agent nodes are designated as [public](/1.10/overview/concepts/#public-agent-node) or [private](/1.10/overview/concepts/#private-agent-node) during installation. By default, they are designated as private during [GUI][1] or [CLI][2] installation.
 
 ### Prerequisites:
-These steps must be performed on a machine that is configured as a DC/OS node. Any tasks that are running on the node will be terminated during this conversion process.
+The following steps must be performed on a machine that is configured as a DC/OS node. Any tasks that are running on the node will be terminated during this conversion process.
 
 *   DC/OS is installed using the [custom](/1.10/installing/oss/custom/) installation method and you have deployed at least one [master](/1.10/overview/concepts/#master) and one [private](/1.10/overview/concepts/#private-agent-node) agent node.
 *   The archived DC/OS installer file (`dcos-install.tar`) from your [installation](/1.10/installing/oss/custom/gui/#backup).     
 *   The CLI JSON processor [jq](https://github.com/stedolan/jq/wiki/Installation).
 *   SSH installed and configured. This is required for accessing nodes in the DC/OS cluster.
 
+**Note:** Use the following steps to convert your agent node only. You will not be able to actually uninstall DC/OS here. In order to uninstall DC/OS, you must [re-image](https://docs.mesosphere.com/1.11/installing/oss/custom/uninstall/) the operating system on your nodes. The improvement process for uninstalling DC/OS will be documented in later versions.
```

Comment:
Would it be helpful to change the wording here slight from:

> In order to uninstall DC/OS, you must [re-image]

To

> In order to uninstall DC/OS, currently you must [re-image]

and then perhaps:

> The improvement process for uninstalling DC/OS will be documented in later versions.

Could just be

> This may change in future releases of DC/OS

Let me know what you think.